requirements.txt

```diff
@@ -9,6 +9,7 @@ Flask==1.0.2
 Flask-RESTful==0.3.6
 itsdangerous==0.24
 Jinja2>=2.10.1
+luqum==0.8.0
```

Comment:
ah, let me do some research to see if there is an alternative. I wasn't thinking about restrictions on licensing

Reply:
update from my end: I haven't had much time to do more research here, but I will say that this search works great at Square :)

Reply:
Sorry  missed this. LGPL in general is considered to be incompatible with ASLv2. https://apache.org/legal/resolved.html . Apache's stance is sometimes a bit more strict than other projects, however following its guideline will keep us out of muddy waters.

Reply:
I probably won't reimplement the functionality since we are already using it fine in production at Square. But if anyone else sees this and wants to take on this work, they can! :)

Reply:
I think this pr is useful. I have asked the owner of the repo to reconsider dual license.

Reply:
^^ @alran

Reply:
the repo now is duel license :)

Reply:
I will try to reimplement this without that library dependency.

Reply:
oh, amazing. Keep me updated!